Client/Frontend/Browser/Search/InitialSearchEngines.swift

```diff
@@ -68,13 +58,7 @@ class InitialSearchEngines {
     /// List of available engines for given locale. This list is sorted by with priority and default engines at the top.
     var engines: [SearchEngine]
     
-    /// Lists of engines available during onboarding.
-    var onboardingEngines: [SearchEngine] {
-        engines.filter { !$0.id.excludedFromOnboarding(for: locale) }
-    }
-    
-    static let braveSearchOnboardingRegions = ["US", "CA"]
-    static let qwantDefaultRegions = ["FR"]
+    static let braveSearchDefaultRegions = ["US", "CA", "GB", "FR", "DE"]
```

Comment:
@iccub I followed up with @ecnmst on Slack (had the same concern for Android) about whether we should make Brave Search visible in the search onboarding OUTSIDE of US/CA/GB/FR/DE and it seems we want to keep Brave out of the search onboarding list for regions outside of there (for now)

Can we keep the exclude logic in place for now? Basically if the search engine list shows, it can not show Brave Search (yet)

Reply:
yes, im creating followup PR for this

Reply:
Here's the test plan I had for Android https://github.com/brave/brave-browser/issues/18452